README.md

```diff
@@ -89,10 +89,29 @@ You can run the linter with:
 npm run lint
 ```
 
-You can run local functional tests with:
+You can run local functional tests:
+
+We use s3cmd and Redis for our functional tests.
+
+* Install [s3cmd](http://s3tools.org/download)
+* Install [redis](https://redis.io/) and start Redis.
```

Comment:
It would be appropriate to link the users to the download url https://redis.io/download